uiServices/src/homepage.go

```diff
@@ -104,6 +105,7 @@ func routeConfigs() http.Handler {
 	router.Post("/", postConfigs)
 	router.Get("/{configName}", getConfigs)
 	router.Put("/{configName}", putConfigs)
+	router.Put("/filename/{configFileName}/{newConfigFileName}", putConfigFileName)
```

Comment:
not sure i would have gone with this approach for renaming the file i.e. creating a new route